src/lib/feature-flags/__tests__/utils.test.ts

```
@@ -0,0 +1,290 @@
+    });
+  });
+
+  describe('isFeatureFlagEnvironment', () => {
+    it('should return true for valid environments', () => {
+      expect(isFeatureFlagEnvironment()).toBe(true); // development (mocked)
+    });
+  });
+
+  describe('getFeatureFlagDocUrl', () => {
+    beforeEach(() => {
+      delete process.env.NEXT_PUBLIC_APP_URL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)